functions/index.js

```diff
@@ -126,39 +126,60 @@ function handleMessage(sender_psid, received_message) {
 }
 
 function handlePostback(sender_psid, received_postback) {
+
   console.log('ok')
-   let response;
+  var userPublicProfile = "https://graph.facebook.com/v2.6/" + sender_psid + "?fields=first_name,last_name,profile_pic&access_token="+ PAGE_ACCESS_TOKEN;
+  let response;
   let payload = received_postback.payload;
 
-if (payload === 'GET_STARTED_PAYLOAD') {
-    response = {
-      "attachment": {
-        "type": "template",
-        "payload": {
-          "template_type": "generic",
-          "elements": [{
-            "title":"Hi {{user_first_name}}!",
-            "subtitle": "I'm Duber. I can help you find people with answers to your questions.",
-            "buttons": [
-              {
-                "type": "postback",
-                "title": "Complete my Profile",
-                "payload": "PAYLOAD_COMPLETE_MY_PROFILE",
+  if (payload === 'GET_STARTED_PAYLOAD') {
+      let user_first_name 
```

Comment:
Too many spaces before `let`. There should only be two spaces at each level of indentation. so it should like this:
ìf

Reply:
I'm using sublime. What style guide should we stick to. Tabs or spaces? I have changed it to 2 spaces as per this style guide.
https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Reply:
I suggest you switch to visual studio code. there's also a good integration for github. Let's stick to spaces. Also, please address this comment:

```Remove unwanted files under node_modules. These are not being used anywhere. The node files we're using are under functions/nodemodules. Not the node_modules folder at the root. That one should be deleted.```

Reply:
I prefer to use sublime.